mne/viz/utils.py

```diff
@@ -1240,15 +1240,15 @@ def __init__(self, cbar, mappable):
 
     def connect(self):
         """Connect to all the events we need."""
-        self.cidpress = self.cbar.patch.figure.canvas.mpl_connect(
+        self.cidpress = self.cbar.ax.figure.canvas.mpl_connect(
```

Comment:
This is just because `cbar.patch` is deprecated now, and all we need is the figure so we might as well go through `ax` instead